lib/cleaner.rb

```diff
@@ -1,9 +1,16 @@
+require 'CSV'
+
 class Cleaner
-  attr_accessor :contents
+  attr_accessor :merchant_csv,
+                :items_csv
+
+  def initialize(merchant_csv, items_csv)
+    @merchant_csv = merchant_csv
+    @items_csv = items_csv
+  end
```

Comment:
this looks good, but I think we have a way to eliminate the readers/initializers all together. I will merge this and create a new branch with my logic so we can review this afternoon!